Add-on fails to build on jenkins win32

> C:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt\stdio.h(1925): warning C4005: 'snprintf': macro redefinition
c:\jenkins\slave\workspace\win-32\project\cmake\addons\build\libxml2\src\libxml2\config.h(99): note: see previous definition of 'snprintf'
C:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt\stdio.h(1927): fatal error C1189: #error:  Macro definition of snprintf conflicts with Standard Library function declaration

Full build log: http://jenkins.kodi.tv/job/WIN-32/9005/consoleText

--- a/depends/common/libxml2/libxml2.txt
+++ b/depends/common/libxml2/libxml2.txt
@@ -1,1 +1,1 @@
-libxml2 http://xmlsoft.org/sources/libxml2-2.9.3.tar.gz
+libxml2 http://xmlsoft.org/sources/libxml2-2.9.4.tar.gz

--- a/src/XMLTV.cpp
+++ b/src/XMLTV.cpp
@@ -23,6 +23,7 @@
 #include "XMLTV.h"
 
 #include <algorithm>
+#include <iterator>
 
 #include "p8-platform/util/StringUtils.h"
 

